USAGE.md

```diff
@@ -452,6 +452,40 @@ In order to enable Direct debugging for your app, make sure that your App.cpp/Ap
   InstanceSettings.UseWebDebugger = false;
 ```
 
+# Calling methods on XAML objects ("commands")
+Some types support custom commands to expose some functionality of the underlying platform. 
+For example, this allows calling the `MediaPlayerElement.Play()` method in response to some other action, or programatically showing a flyout menu:
+
+```jsx
+  const _tbRef = React.useRef<TextBlock>(null);
+  const _menuRef = useRef<MenuFlyout>(null);
+
+  const [x, setX] = React.useState(100);
+  // ...
+  return
+<TextBox
+  text={x}
+  onBeforeTextChanging={e => {
+    setX(e.nativeEvent.args.newText);
+}}
+/>
+<TextBlock
+  text="Hello"
+  onTapped={e => {
+    MenuFlyout.ShowAt(_menuRef, {point: {x: x, y: 42}});
+  }}
+  ref={t => {
```

Comment:
Is it possible to wire this up so that passing in the ref holder directly works?

Reply:
that would be ideal. I'll file a bug to fix this later, this change is getting too large already : )